package.json

```diff
@@ -29,7 +29,7 @@
     "changeset": "changeset && pnpm install",
     "clean": "pnpm --recursive --parallel clean",
     "clean:nodemodules": "find . -type d -name \"node_modules\" -exec rm -fr {} +",
-    "dev": "pnpm --recursive --parallel dev",
```

Comment:
I removed these packages, so they don't start the dev server with all packages, we can remove that filter again though :)